app/controllers/categories_controller.rb

```diff
@@ -15,14 +14,17 @@ def show
 
     respond_with @category do |format|
       if @user
-        format.html { 
-          render :template => 'users/show',
-                 :layout   => 'users'
-        }
+        format.html { render :template => 'users/show', :layout   => 'users' }
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.